channel.go

```diff
@@ -104,6 +104,10 @@ func (q queryResponse) updateChannel(ch *Channel) {
 	}
 }
 
+type SendBulkMessagesResponse struct {
+	Messages []Message `json:messages`
```

Comment:
I think it should be json:"messages", like the linter suggests

Reply:
ouch :D